hw/top_darjeeling/data/top_darjeeling.hjson

```diff
@@ -293,8 +293,6 @@
       // the ip.hjson will declare the reset port names
       // If none are defined at ip.hjson, rst_ni is used by default
       reset_connections: {rst_ni: "lc_io_div4"},
-      // this module's interrupts sent to this PLIC
```

Comment:
Thanks for making this fixup commit. I think it looks really good: let's squash it with the first commit?